smriprep/workflows/surfaces.py

```diff
@@ -268,36 +234,131 @@ def init_surface_recon_wf(*, omp_nthreads, hires, name="surface_recon_wf"):
+        )
+
+        # fmt:off
+        workflow.connect([
+            (inputnode, fsnative2t1w_xfm, [('t1w', 'target_file')]),
+            (autorecon1, fsnative2t1w_xfm, [('T1', 'source_file')]),
+            (fsnative2t1w_xfm, outputnode, [('out_reg_file', 'fsnative2t1w_xfm')]),
+        ])
+        # fmt:on
+
+    return workflow
+
+
+def init_refinement_wf(*, name="refinement_wf"):
+    r"""
+    Refine ANTs brain extraction with FreeSurfer segmentation
+
+    Workflow Graph
+        .. workflow::
+            :graph2use: orig
+            :simple_form: yes
+
+            from smriprep.workflows.surfaces import init_refinement_wf
+            wf = init_refinement_wf()
+
+    Inputs
+    ------
+    subjects_dir
+        FreeSurfer SUBJECTS_DIR
+    subject_id
+        FreeSurfer subject ID
+    fsnative2t1w_xfm
+        LTA-style affine matrix translating from FreeSurfer-conformed subject space to T1w
+    reference_image
+        Input
+    t2w
+        List of T2-weighted structural images (only first used)
+    flair
+        List of FLAIR images
+    skullstripped_t1
+        Skull-stripped T1-weighted image (or mask of image)
+    ants_segs
+        Brain tissue segmentation from ANTS ``antsBrainExtraction.sh``
+    corrected_t1
+        INU-corrected, merged T1-weighted image
+    subjects_dir
+        FreeSurfer SUBJECTS_DIR
+    subject_id
+        FreeSurfer subject ID
+
+    Outputs
+    -------
+    subjects_dir
+        FreeSurfer SUBJECTS_DIR
+    subject_id
+        FreeSurfer subject ID
+    t1w2fsnative_xfm
+        LTA-style affine matrix translating from T1w to FreeSurfer-conformed subject space
+    fsnative2t1w_xfm
+        LTA-style affine matrix translating from FreeSurfer-conformed subject space to T1w
+    out_brainmask
+        Refined brainmask, derived from FreeSurfer's ``aseg`` volume
+
+    See also
+    --------
+    * :py:func:`~smriprep.workflows.surfaces.init_autorecon_resume_wf`
+
+    """
+    workflow = Workflow(name=name)
+    workflow.__desc__ = """\
+Brain surfaces were reconstructed using `recon-all` [FreeSurfer {fs_ver},
+RRID:SCR_001847, @fs_reconall], and the brain mask estimated
+previously was refined with a custom variation of the method to reconcile
+ANTs-derived and FreeSurfer-derived segmentations of the cortical
+gray-matter of Mindboggle [RRID:SCR_002438, @mindboggle].
+""".format(
+        fs_ver=fs.Info().looseversion() or "<ver>"
+    )
+
+    inputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=[
+                "reference_image",
+                "ants_segs",
+                "fsnative2t1w_xfm",
+                "subjects_dir",
+                "subject_id",
+            ]
+        ),
+        name="inputnode",
+    )
+    outputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=[
+                "out_brainmask",
+            ]
+        ),
+        name="outputnode",
+    )
+
+    aseg_to_native_wf = init_segs_to_native_wf()
```

Comment:
Duplicated here and in `init_surface_derivatives_wf`. Acceptably small duplication of effort for conceptual clarity, IMO.